src/lint/lint.ts

```diff
@@ -320,8 +320,9 @@ async function fix(
       await applyEdits(edits, makeParseLoader(analyzer, analysis));
 
   for (const [newPath, newContents] of editedFiles) {
+    const filePath = newPath.replace('file://', '').replace(config.root, '');
```

Comment:
this is ugly, I'm probably missing something, but makes the --fix work

Reply:
We need a common utility for this, but something like:

```ts
    const url = new URL(newPath);
    let documentPath = decodeURIComponent(documentURL.pathname);
    if (isWindows() && documentPath.startsWith('/')) {
      documentPath = documentPath.substring(1);
    }
```

Reply:
I moved this url resolution in `util`, method `resolvePath`. I'm using https://www.npmjs.com/package/url-js for getting the `URL` object, as it's a minimal package with no deps.